bugbug/utils.py

```diff
@@ -513,3 +513,16 @@ def escape_markdown(text: str) -> str:
         .replace(")", "\\)")
         .replace("|", "\\|")
     )
+
+
+# Remove business days from date range
+
+
+def business_day_range(end_date, start_date):
```

Comment:
I would name the function something more descriptive like `get_business_days_count()`. Also, I expect to have the `start_date` as the first argument.

Reply:
I'll go ahead and switch both of those. Initially I wanted to keep things the same order as the original subtraction but no, it doesn't really make sense here.